cloud/services/object_storage_buckets.go

```diff
@@ -132,14 +132,21 @@ func createObjectStorageKey(ctx context.Context, bucketScope *scope.ObjectStorag
 	return key, nil
 }
 
-func DeleteObjectStorageKeys(ctx context.Context, bucketScope *scope.ObjectStorageBucketScope, logger logr.Logger, keys [2]float64) error {
-
-	for _, keyID := range keys {
-		logger.Info(fmt.Sprintf("revoking object storage access key %s", keyID))
-		if err := bucketScope.LinodeClient.DeleteObjectStorageKey(ctx, int(keyID)); err != nil {
-			return fmt.Errorf("revoke object storage key: %w", err)
+func RevokeObjectStorageKeys(ctx context.Context, bucketScope *scope.ObjectStorageBucketScope, keyIDs [2]int, logger logr.Logger) error {
+	for _, keyID := range keyIDs {
+		if err := revokeObjectStorageKey(ctx, bucketScope, keyID, logger); err != nil {
+			logger.Info("Failed to revoke object storage key", "id", keyID, "error", err.Error())
 		}
-		logger.Info("revoked object storage key", "key-id", keyID)
```

Comment:
Since this is not a k8s resources, we don't have any indication to know if this was successfully deleted/revoked if we remove this log. Can we put this back in?